app/components/PercentageStats/index.tsx

```diff
@@ -1,35 +1,32 @@
 import React from 'react';
 import {
-    ContainerCard,
-    CompactInformationCard,
+    ContainerCard, NumberOutput,
 } from '@the-deep/deep-ui';
 import { _cs } from '@togglecorp/fujs';
 
 import styles from './styles.css';
 
-export declare type SpacingTypes = 'none' | 'compact' | 'comfortable' | 'loose';
-
 interface PercentageStatsProps {
     className?: string;
+    icon: React.ReactNode;
     heading?: React.ReactNode;
     headerDescription?: React.ReactNode;
     headingSize?: 'large' | 'small' | 'extraSmall' | 'medium' | 'extraLarge' | undefined;
-    StatValue: number;
-    StatValueSpacing?: SpacingTypes | undefined;
-    StatValueDescription: string | undefined;
-    StatValueIcon?: React.ReactNode;
+    suffix?: string;
```

Comment:
```suggestion
    suffix?: 'percentage' | 'something';
```

If we are only expecting percentage, add isPercent prop instead of suffix.

Reply:
i have implemented as you have suggested